packages/inline-definition/src/InlineDefinition.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Tooltip, { TooltipProps } from '@leafygreen-ui/tooltip';
+import { Body } from '@leafygreen-ui/typography';
+import { css, cx } from '@leafygreen-ui/emotion';
+import { uiColors } from '@leafygreen-ui/palette';
+
+const underline = css`
+  display: inline-flex;
```

Comment:
What are we using this display property for?

Reply:
Now that we're using background-position: center bottom, it's no longer necessary